### PR TITLE
[Flutter-Parent][MBL-13624] Not-A-Parent screen

### PR DESCRIPTION
**Pull request by JordanMarshall**
Also adds a basic splash screen whose architecture has purposefully been kept simple so as to reduce churn when the requirements of the screen evolve (masquerade check, locale change, etc) and a more opinionated architecture is called for.

Additional minor changes:
 - Moves static colors from ParentTheme to ParentColors
 - Makes Tweaks to the dashboard appbar UI
 - Adds a DropdownArrow widget that matches the designs
 - Adds an optional button to EmptyPandaWidget
 - Uses provider in ParentTheme to notify DefaultParentTheme of theme state changes
 - Updates Avatar widget to default the background color to nearSurfaceColor
 - Updates Avatar widget to use CachedNetworkImage

### CODE REVIEW

**Comment by instructure-cx** [comment]
Scan submitted to Checkmarx

**Comment by instructure-cx** [comment]
### Checkmarx scan completed
[Full Scan Details](https://cxmgr.inssec.info/CxWebClient/ViewerMain.aspx?scanid=1014302&projectid=20)
#### Checkmarx Scan Summary
Severity|Count
---|---
High|0
Medium|0
Low|0
Informational|0

#### Violation Summary
Severity|Count
---|---

File: apps/flutter_parent/assets/svg/canvas-logo.svg [hunk 1]
```diff
@@ -0,0 +1,3 @@
+<svg width="64" height="64" viewBox="0 0 64 64" fill="none" xmlns="http://www.w3.org/2000/svg">
```
Comment (StephenBrough) [inline apps/flutter_parent/assets/svg/canvas-logo.svg]: Do these SVGs need license?

File: apps/flutter_parent/lib/l10n/app_localizations.dart [hunk 2]
```diff
@@ -319,7 +319,8 @@ class AppLocalizations {
 
   String get noCoursesTitle => Intl.message('No Courses', desc: 'Title for having no courses');
 
-  String get noCoursesMessage => Intl.message("Your child’s courses might not be published yet.", desc: 'Message for having no courses');
```
Comment (StephenBrough) [inline apps/flutter_parent/lib/l10n/app_localizations.dart]: Child -> Student

File: apps/flutter_parent/lib/utils/common_widgets/canvas_loading_indicator.dart [hunk 3]
```diff
@@ -0,0 +1,152 @@
+// Copyright (C) 2019 - present Instructure, Inc.
```
Comment (StephenBrough) [inline apps/flutter_parent/lib/utils/common_widgets/canvas_loading_indicator.dart]: I can't believe you've done this.

File: apps/flutter_parent/lib/utils/common_widgets/dropdown_arrow.dart [hunk 4]
```diff
@@ -0,0 +1,61 @@
+// Copyright (C) 2019 - present Instructure, Inc.
```
Comment (StephenBrough) [inline apps/flutter_parent/lib/utils/common_widgets/dropdown_arrow.dart]: I can't believe you've done _this._

**Pull request merged**